tests/integration/conftest.py

```diff
@@ -570,11 +570,13 @@ async def tmate_ssh_server_unit_ip_fixture(
     tmate_ssh_server_app: Application,
 ) -> bytes | str:
     """tmate-ssh-server charm unit ip."""
-    status: FullStatus = await model.get_status([tmate_ssh_server_app.name])
+    app_name = tmate_ssh_server_app.name
+    status: FullStatus = await model.get_status([app_name])
+    app_status = status.applications[app_name]
+    assert app_status is not None, f"Application {app_name} not found in status"
```

Comment:
Are these changes intended as part of this PR?

Reply:
required to pass linting (something changed in the pylib-juju)